src/sass/components/_customersreview.scss

```diff
@@ -0,0 +1,64 @@
+.customers-img {
+  padding: 60px 97px 32px 98px;
```

Comment:
Wysentruj obrazek w poziomie za pomocą np flexa, a na dole margin odpychający recenzję

Reply:
Też, o tym myślałem, ale czemu moim zdaniem te rozwiązanie ma sens, nie wiem czy przez to nie będę musiał ubrać <img> w jakiegoś diva